src/js/api.js

```diff
@@ -64,10 +64,14 @@ module.exports = {
    return postHandler(url, cb, '/submissions');
  },
 
- getEdits: function(url, cb){
+ getEditsByType: function(url, cb){
    return getHandler(url, cb, '/edits');
  },
 
+ getEditsByRow: function(url, cb){
```

Comment:
There actually is already an entry, but in checking it out I realized there is an error, so I'll fix that as well ;)